tests/mail/tests.py

```diff
@@ -1552,6 +1552,25 @@ def test_all_params_can_be_set_before_send(self):
         self.assertIs(email.get_connection(), new_connection)
         self.assertNotIn("original", message.as_string())
 
+    def test_unicode_attachment_encoding(self):
```

Comment:
Could you move this test up just after `test_attach_non_utf8_text_as_bytes()` (around line 915) to keep it near the other attachment-content tests?